spec/requests/jobseekers/job_applications_spec.rb

```diff
@@ -33,6 +33,16 @@
           expect(get(new_jobseekers_job_job_application_path(vacancy.id))).to redirect_to(jobseekers_job_applications_path)
         end
       end
+
+      context "when a non-draft job application already exists" do
+        let!(:job_application) { create(:job_application, :status_submitted, jobseeker: jobseeker, vacancy: vacancy) }
+        let(:new_vacancy) { create(:vacancy, organisation_vacancies_attributes: [{ organisation: build(:school) }]) }
+
+        it "redirects to `new_quick_apply_jobseekers_job_job_application_path`" do
```

Comment:
rather than the path in rails speak, would it be better to describe in english where its redirecting to ?

Reply:
Hmm. My thinking is... controllers and request specs don't care much about humans or english, but happy path (system) specs, care very much about humans and english https://github.com/DFE-Digital/teaching-vacancies/blob/2437b7bf2d60685c4ab0cdbd61e9b22726c02a47/spec/system/jobseekers_can_quick_apply_for_a_vacancy_spec.rb#L15